### DISCUSSION TITLE
[ FF7 ] Crash after field battle

### DISCUSSION THREAD

**Discussion by driver9** [julianxhokaxhiu/FFNx#223] [Q&A] [answered]
Hello got a crash after a battle on the field and the game returned me to the shinra building and crashed...is my save file lost now? thanks for the help..

[crash files.zip](https://github.com/julianxhokaxhiu/FFNx/files/6176032/crash.files.zip)

**Comment by julianxhokaxhiu** [comment, marked as answer]
The log actually seems to mention your session was clean. If you're using a mod manager please approach the relative author and ask for help using the related channels. Thank you.
